knowledge-base/kb-panelbar-remove-mouseover-and-selected-overlay-child-item.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Remove MouseOver and Selected Overlay on Child Item
+description: Avoid mouse enter and selection backgrounds for child RadPanelBarItem when it selected or hovered
+type: how-to
+page_title: Clear MouseOver and Selected Backgrounds from Child Item
+slug: kb-panelbar-remove-mouseover-and-selected-overlay-child-item
+position: 0
+tags: hover,selection,overlay,background
+ticketid: 1401889
+res_type: kb
+---
+
+## Environment
+<table>
+	<tr>
+		<td>Product Version</td>
+		<td>2019.1.220</td>
+	</tr>
+	<tr>
+		<td>Product</td>
+		<td>RadPanelBar for WPF</td>
+	</tr>
+</table>
+
+## Description
+
+How to remove mouse over and selected background colors of a child RadPanelBarItem when it is hovered or selected.
+
+## Solution
+
+Extract and modify the content of the Template property of RadPanelBarItem.
+
+__To remove the mouse over overlay:__
+
+1. Extract the Style of the RadPanelBarItem. 
+2. Find the Setter that sets the Template property.
```

Comment:
We can elaborate on this step or add a separate step that specifies that we need to find the RadPanelBarItemTemplate ControlTemplate and the look for the visual state in it.